### PR TITLE
Create traidnt.net.xml

### PR DESCRIPTION
**Pull request by ihave100openPRs**

### CODE REVIEW

File: src/chrome/content/rules/traidnt.net.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<ruleset name="traidnt.net">
+	<target host="traidnt.net" />
+	<target host="www.traidnt.net" />
```
Comment (gloomy-ghost) [inline src/chrome/content/rules/traidnt.net.xml]: ads.traidnt.net

File: src/chrome/content/rules/traidnt.net.xml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+	<target host="dir.traidnt.net" />
+	<target host="news.traidnt.net" />
```
Comment (gloomy-ghost) [inline src/chrome/content/rules/traidnt.net.xml]: privacy.traidnt.net

**Comment by ihave100openPRs** [comment]
@gloomy-ghost  Added

**Comment by gloomy-ghost** [comment]
merged, thanks!

**Pull request merged**